try:
	print(num)
	print("------1------")
except NameError:
	print("如果捕获到异常后做的 处理...")


print("------2-------")


输出结果>>>

如果捕获到异常后做的 处理....
-------2---------



说明, print(num)捕获异常后, 后面代码不执行